test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,95 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.*;
+
+import java.util.Collections;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Split the import into multiple groups as the packages are imported from different sources.